bmorph/util/mizuroute_utils.py

```diff
@@ -570,8 +587,8 @@ def map_ref_sites(routed: xr.Dataset, gauge_reference: xr.Dataset,
         fill_up_isegs = np.where(np.isnan(routed['up_ref_seg'].values))[0]
         fill_down_isegs = np.where(np.isnan(routed['down_ref_seg'].values))[0]
 
-        routed['kge_up_gauge'] = 0 * routed['is_gauge']
-        routed['kge_down_gauge'] = 0 * routed['is_gauge']
+        routed['kge_up_gauge'] = min_kge * routed['is_gauge']
+        routed['kge_down_gauge'] = min_kge * routed['is_gauge']
```

Comment:
`is_gauge` is a boolean, and if it's False, then the value will be set to zero. This doesn't matter for the previous version since we were setting it to zero anyways, but this improvement may not be universal in its current state.